src/components/InvoicesList/InvoicesList.js

```diff
@@ -0,0 +1,119 @@
+  const [stateInvoices, setStateInvoices] = useState({ loading: true });
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setStateInvoices({ loading: true });
+      const invoicesResponse = await dopplerBillingApiClient.getInvoices(0, 0);
+      if (!invoicesResponse.success) {
+        setStateInvoices({ loading: false, items: [], success: false });
+      } else {
+        setStateInvoices({
+          loading: false,
+          success: true,
+          items: invoicesResponse.value.items,
+          totalItems: invoicesResponse.value.totalItems,
+        });
+      }
+    };
+
+    fetchData();
+  }, [dopplerBillingApiClient]);
+
+  return (
+    <>
+      <Helmet>
+        <title>{_('invoices_list.title')}</title>
+        <meta name="invoices" />
+      </Helmet>
+      <HeaderSection>
+        <div className="col-sm-12 col-md-12 col-lg-12">
+          <h2>
+            <FormattedMessage id="invoices_list.title" />
+          </h2>
+          <p>
+            <FormattedMessage id="invoices_list.sub_title" />
+          </p>
+        </div>
+      </HeaderSection>
+      <section className="dp-container">
+        <div className="dp-table-responsive">
+          {stateInvoices.loading ? (
+            <Loading page />
+          ) : !stateInvoices.success ? (
+            <div class="dp-msj-error dpsg-slow-animation bounceIn">
+              <p>
+                <FormattedMessage id="invoices_list.error_msg" />
+              </p>
+            </div>
+          ) : (
+            <table className="dp-c-table">
+              <thead>
+                <tr>
+                  <th scope="col">
+                    <FormattedMessage id="invoices_list.account_id_column" />
+                  </th>
+                  <th scope="col">
+                    <FormattedMessage id="invoices_list.product_column" />
+                  </th>
+                  <th scope="col">
+                    <FormattedMessage id="invoices_list.date_column" />
+                  </th>
+                  <th scope="col">
+                    <FormattedMessage id="invoices_list.currency_column" />
+                  </th>
+                  <th scope="col">
+                    <FormattedMessage id="invoices_list.amount_column" />
+                  </th>
+                  <th scope="col"></th>
+                </tr>
+              </thead>
+              <tbody>
+                {stateInvoices.totalItems > 0 ? (
+                  <>
+                    {stateInvoices.items.map((invoice, index) => {
+                      return (
+                        <tr key={index}>
+                          <td>{invoice.accountId}</td>
+                          <td>{invoice.product}</td>
+                          <td>
+                            <Moment format="DD/MM/YYYY">{invoice.date}</Moment>
```

Comment:
@fcoronelmakingsense If I am not wrong we are formatting dates without using Moment, could you share us an example?

Reply:
I saw in the code and the team is using **FormattedDate**. Is it correct?

Reply:
I used the FormattedDate and it looks fine:

![image](https://user-images.githubusercontent.com/70591946/94691948-5542e080-0308-11eb-8ca1-b01908a4091b.png)